docs/live_update_reference.md

```diff
@@ -0,0 +1,22 @@
+---
+title: Live Update Reference
+layout: docs
+---
+##### (This doc provides the technical specifications of Tilt's `LiveUpdate` functionality. For a tutorial that walks you through a sample project, see [Optimizing a Tiltfile](live_update_tutorial.html).)
+
+When specifying how to build an image (via `docker_build()` or `custom_build()`), you may optionally pass the `live_update` arg.
+
+`live_update` takes a list of `LiveUpdateSteps` that tell Tilt how to update a running container in place. (This allows us to skip Docker build/push, take advantages of on-disk caches, and generally give you a faster development experience.)
```

Comment:
A couple of things that might be useful and/or repetitive here:
1. "update a running container in place instead of paying the cost of building a new image and redeploying"
2. a "See here for details" link to a blog post - or maybe we're waiting for a new post from DB that's more live_update than fast_build?

Reply:
ah yep, waiting on a blog post